packages/trader/src/AppV2/Utils/positions-utils.ts

```diff
@@ -52,7 +52,7 @@ export const getProfit = (
     contract_info: TPortfolioPosition['contract_info'] | TClosedPosition['contract_info']
 ): string | number => {
     return (
-        (contract_info as TClosedPosition['contract_info']).profit_loss ??
+        (contract_info as TClosedPosition['contract_info']).profit_loss?.replaceAll(',', '') ??
```

Comment:
this is fixing an issue with 0.00 USD displayed in Postions/Closed tab instead of big profit values with comma.